coldfront/plugins/qumulo/management/commands/add_qumulo_allocation_attribute_type.py

```diff
@@ -126,3 +126,95 @@ def handle(self, *args, **options):
             is_changeable=True,
             is_unique=False,
         )
+
+        # jprew - here is where I'm adding ITDEV-35273-related attribute types
+        # for the allocation collection
+        AllocationAttributeType.objects.get_or_create(
+            attribute_type=AttributeType.objects.get(name="Yes/No"),
+            name="secure",
+            is_required=True,
+            is_private=False,
+            is_changeable=False,
+        )
+
+        # indicates whether the allocation is subject to being audited
+        AllocationAttributeType.objects.get_or_create(
+            attribute_type=AttributeType.objects.get(name="Yes/No"),
+            name="audit",
+            is_required=True,
+            is_private=False,
+            is_changeable=False,
+        )
+
+        AllocationAttributeType.objects.get_or_create(
```

Comment:
@peaches7083 [MAYOR] duplicate of the above.

Reply:
Good catch, sorry about that.

Reply:
Removed.